package.json

```diff
@@ -6,7 +6,7 @@
   },
   "filename": "brainstem",
   "standalone": "Brainstem",
-  "version": "0.4.16",
+  "version": "0.4.17-a",
```

Comment:
We should make this non-alpha once this has been completely tested. Can be in a separate PR to make it easier, but you can publish versions outside of merging PRs.